### PR TITLE
Fix gettext support

### PR DESCRIPTION
**Pull request by hauleth**

### CODE REVIEW

**Comment by simonprev** [comment]
Thank you for the pull request 😄 Could you provide a test that validate your modification on the parser?

PS: Your modification of the "test loop" is awesome and should be ported to every other formatter tests 😉

**Comment by hauleth** [comment]
Sure.

**Comment by hauleth** [comment]
Tests written.

File: test/langue/gettext/formatter_test.exs [hunk 1]
```diff
@@ -13,19 +20,25 @@ defmodule AccentTest.Formatter.Gettext.Parser do
     Simple
   ]
 
-  test "gettext" do
-    Enum.each(@tests, fn ex ->
-      {expected_parse, result_parse} = FormatterTestHelper.test_parse(ex, Parser)
-      {expected_serialize, result_serialize} = FormatterTestHelper.test_serialize(ex, Serializer)
+  for ex <- @tests do
+    test "getext #{ex}" do
```
Comment (simonprev) [inline test/langue/gettext/formatter_test.exs]: There is a typo: `getext` should be `gettext`

File: test/plugs/movement_context_parser_test.exs [hunk 2]
```diff
@@ -74,7 +74,7 @@ defmodule AccentTest.Plugs.MovementContextParser do
       |> assign(:project, project)
       |> MovementContextParser.call([])
 
-    assert conn.assigns[:document_parser] == (&Langue.Formatter.Json.Parser.parse/1)
+    assert conn.assigns[:document_parser] == &Langue.Formatter.Json.Parser.parse/1
```
Comment (simonprev) [inline test/plugs/movement_context_parser_test.exs]: Same as your other PR: `mix format` should use Elixir 1.6.4 locally to pass on Travis (which run 1.6.4)

**Pull request merged**